astropy/wcs/tests/extension/wcsapi_test.c

```diff
@@ -49,8 +47,7 @@ PyInit_wcsapi_test(void)
   m = PyModule_Create(&moduledef);
 
   if (m == NULL) {
-      printf("HERE\n");
```

Comment:
Not sure if that was essential ... I guess not, so I removed it.

Reply:
Added by @mdboom 4 years ago... But I agree that this can be removed.